featurebyte/worker/task/target_table.py

```diff
@@ -116,7 +114,8 @@ async def execute(self, payload: TargetTableTaskPayload) -> Any:
                 request_input=payload.request_input,
                 primary_entity_ids=primary_entity_ids,
                 purpose=purpose,
-                has_row_index=True,
```

Comment:
On second thought, I think we want this to be True since we want the created observation table to be able to benefit from caching when used. We should still call the `add_row_index_column` function above when `observation_set` is not a row index eligible observation table.

Reply:
I will add it